search_action_domain/README.rst

```diff
@@ -5,7 +5,7 @@ Search Action Domain
 ====================
 
 This module lets the user to use server actions as a filters in the "Favorites"
-section. In other words, the user might use complex queries to search the records.  
+section. In other words, the user might use complex queries to search the records.
```

Comment:
Apply this in a separated commit, as it's a different module. This same PR so we can see it's green